### ISSUE TITLE
Question: Is there any way to change the delay between comments/replies?

### ISSUE THREAD

**Issue by Alex-Sour** [charlypoirier/redditube#4]
When a comment is read, it takes way too long for the next one imo.

**Comment by charlypoirier** [comment]
I agree, I remember noticing this as well and couldn't find a way to make it faster.
I'll keep this issue open until I have a solution for this.

**Comment by Alex-Sour** [comment]
Alright

**Comment by Alex-Sour** [comment]
Instead of a faster transition there could be a way to configure it and choose a delay in ms

**Comment by willuhm-js** [comment]
What is the cause of the delay?
